cli/crates/backend/assets/default-schema.graphql

```diff
@@ -1,11 +1,43 @@
-type TodoList @model {
-  id: ID!
+# Welcome to Grafbase!
+# Build your own database, integrate auth, permission rules, custom resolvers, search, and more with Grafbase.
+
+# Configure Auth
+# https://grafbase.com/docs/auth
+#
+# schema @auth(providers: [{ type: oidc, issuer: "{{ env.ISSUER_URL }}" }], rules: [{ allow: private }]) {
+#   query: Query
+# }
+
+# Define Data Model(s)
+# https://grafbase.com/docs/database
+type Post @model @search {
   title: String!
-  todos: [Todo]
+  slug: String! @unique
+  content: String
+  publishedAt: DateTime
+  comments: [Comment]
+  likes: Int @default(value: 0)
+  tags: [String] @length(max: 5)
+  author: User
 }
 
-type Todo @model {
-  id: ID!
-  title: String!
-  complete: Boolean!
+type Comment @model @search {
+  post: Post!
+  body: String!
+  likes: Int @default(value: 0)
+  author: User
 }
+
+type User @model {
+  name: String!
+  email: Email
+  posts: [Post]
+  comments: [Comment]
+  # Extend models with resolvers
+  # https://grafbase.com/docs/edge-gateway/resolvers
+  # gravatar: URL @resolver(name: "user/gravatar")
+}
+
+# Start your backend
```

Comment:
Could we somehow format this so it doesn't all seem on the same level? The URL is essentially a comment and the header could probably be distinguished